tests/pages/account.test.js

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+import { Selector } from 'testcafe';
+
+const el = Selector(selector => document.querySelector(selector));
+
+const url = 'http://localhost:8081';
```

Comment:
I would put this as http://localhost:8081/account

I know you have to log in but this will redirect you there. And will make it easier to see which page is being tested.